cache/lru_cache.cc

```diff
@@ -458,6 +458,8 @@ void LRUCacheShard::Promote(LRUHandle* e) {
         e->Unref();
         e->Free();
         e = nullptr;
+      } else {
+        PERF_COUNTER_ADD(block_cache_standalone_handle_count, 1);
```

Comment:
Should we also have a counter for number of handles converted form standalone to proper entries?

Reply:
There are two type of real handles from secondary cache:
1. standalone handles, with a dummy handle inserted into primary cache.
2. real handles that are inserted into primary cache.

secondary_cache_hit_count = # of standalone handles + # of real handles.
I can add "# of real handles".